webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleAdvancedFilterDrawer.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+  <exo-drawer
+      ref="peopleAdvancedFilterDrawer"
+      id="peopleAdvancedFilterDrawer"
+      right
+      @closed="close">
+    <template slot="title">
+      <span class="popupTitle">{{ $t('pepole.advanced.filter.title') }}</span>
+    </template>
+    <template slot="content">
+      <v-form class="pa-2 ms-2 mt-4">
+      <div class="d-flex flex-column flex-grow-1">
+      <div class="d-flex flex-row" v-for="item in settings" :key="item">
+        <input
+            v-model="item.valueToSearch"
+            :placeholder="getResolvedName(item)"
+            type="text"
+            class="input-block-level ignore-vuetify-classes my-3">
```

Comment:
indentation of two spaces instead of 4

```suggestion
          v-model="item.valueToSearch"
          :placeholder="getResolvedName(item)"
          type="text"
          class="input-block-level ignore-vuetify-classes my-3">
```

Reply:
fixed